samples/GuestConfiguration/installed-application-linux/azurepolicyset.definitions.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/fee5cb2b-9d9b-410e-afe3-2902d90d0004"
+    },
+    {
+        "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/4d1c04de-2172-403f-901b-90608c35c721",
+        "parameters": {
+            "applicationName": {
+                "value": "Application name"
```

Comment:
>Application name [](start = 26, length = 16)

[parameters('applicationName')]

Reply:
Fixed.

Reply:
Still needs to be updated in samples/GuestConfiguration/installed-application-linux/azurepolicyset.definitions.json

---
In reply to: [263165361](https://github.com/Azure/azure-policy/pull/326#discussion_r263165361) [](ancestors = 263165361)

Reply:
Fixed.